src/WindowManager.vala

```diff
@@ -2427,6 +2427,14 @@ namespace Gala {
             new_parent.add_child (actor);
             actor.unref ();
         }
+
+        public void show_window_spread (HashTable<string,Variant>? hints = null) {
+            if (window_overview.is_opened ()) {
+                window_overview.close ();
+            }
+
+            window_overview.open (hints);
+        }
```

Comment:
Maybe better would be to make `window_overview` public, and move this method to the DesktopIntegration?

Reply:
Right thank you!